chain-abci/src/app/query.rs

```diff
@@ -180,6 +179,32 @@ impl<T: EnclaveProxy> ChainNodeApp<T> {
                     }
                 }
             }
+            "validators" => {
```

Comment:
In crypto-com terminology, it's called "council nodes"?

Reply:
The convention is "council node" info =  tendermint validator + custom Chain staking stuff
"validator" info = TM validator info only.
so in this case, I guess it's the former

Reply:
Changed.